services/common-features.yaml

```diff
@@ -32,3 +32,52 @@ features:
     description: |
       Publishes events for creation, deletion, and modification of
       objects in a way that enables users to trigger actions in response.
+  - id: CCC.F08
+    title: Multi-zone Deployability
+    description: |
+      Proving the ability for the service to be deployed in multiple availability
+      zones within a region to increase availability and fault tolerance.
+  - id: CCC.F09
+    title: Monitoring
+    description: |
+      Providing the ability to continuously observe, track, and analyze
+      the performance, availability, and health of the service resources or applications.
+  - id: CCC.F10
+    title: Logging
+    description: |
+      Providing the ability to transmit system events, application activities, and/or
+      user interactions to a logging service
+  - id: CCC.F11
+    title: Backup
+    description: |
+      Providing the ability to create copies of associated data or configurations in form of automated backups,
+      snapshot-based backups, and/or incremental backups.
+  - id: CCC.F12
+    title: Recovery
+    description: |
+      Providing the ability to restore data, system, or application to a functional state
+      after an incident such as data loss, corruption or a disaster.
+  - id: CCC.F13
+    title: Infrastructure as Code
+    description: |
+      Providing the automated deployment of the service resources through templates.
+  - id: CCC.F14
+    title: API Access
+    description: |
+      Allowing users to interact programmatically with the service and its resources using APIs, SDKs and CLI.
+  - id: CCC.F15
+    title: Cost Management
+    description: |
+      Providing the ability to filter spending and to detect cost anomalies by the service.
+  - id: CCC.F16
+    title: Budgeting
+    description: |
+      Providing the ability to set a maximum spend for the service.
```

Comment:
Most of the budget tools have some kind of threshold levels based on expected spend. Do all of them have a "you can't spend more than this" type of thing? I'm not sure!